web/themes/custom/vote_gov/templates/node/node--state-territory--full.html.twig

```diff
@@ -8,25 +8,38 @@
 {% extends 'node.html.twig' %}
 
 {% block content %}
-    <h3>{{ 'How to register to vote in @state_name' | t({ '@state_name': label | render }) }}</h3>
-    <p>{{ 'Start your online registration on <a href="@link">@state_name election website</a>.' | t({
+    {# How to register to vote #}
+    <h2>{{ 'How to register to vote in @state_name' | t({ '@state_name': label | render }) }}</h2>
+
+    <p>{{ 'Start your online registration on <a href="@link">@state_name\'s election website</a>.' | t({
```

Comment:
We need to wrap non-required field output in a condition so that when the value is missing the content will not display.
An example:
```
{% if content.field_registration_link | render %}
// Show the content
{% endif %}
```